openapi/components/schemas/Subscription/SubscriptionReactivation.yaml

```
@@ -40,6 +40,14 @@ properties:
       it is ignored and the next renewal occurs as scheduled.
     type: string
     format: date-time
+  paymentInstrumentId:
+    writeOnly: true
+    description: |-
+      ID of the payment instrument. If this field is omitted, the subscription payment instrument remains unchanged.
```

nitpick: [semanic line breaks](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#line-breaks) (sorry, I should have spotted this earlier)
```suggestion
      ID of the payment instrument. 
      If this field is omitted, the subscription payment instrument remains unchanged.
```